build.sbt

```diff
@@ -1,6 +1,6 @@
 import scala.collection.JavaConverters._
 
-name := """nakadi-klients"""
+name := "nakadi-klients"
```

Comment:
Activator places three-quotes but there's no reason for that. (three-quotes is useful e.g. in defining regex strings)